src/objects.js

```diff
@@ -1,41 +1,66 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {
```

Comment:
this could be

`````
return {
    name,
    age
 }
``````
The properties have the same names as the variables. The object literal property value shorthand was introduced in ES6 to shorten the object initialisation.It allows us to define an object whose keys have the same names as the variables passed in as properties by simply passing the variables. The property value shorthand syntax automatically converts each variable to a key-value pair with the variable name as a property key and the variable value as a property value.